docs/install/containerd/containerd-install.md

```diff
@@ -0,0 +1,181 @@
+# Install Kata Containers with containerd
+
+> **Warning:**
+>
+> These instructions install containerd and Kata Containers from binary
+> release packages. It is your responsibility to ensure these packages are
+> kept current as new versions are released.
+>
+> To check for the latest version of Kata Containers,
+> run `kata-runtime kata-check --only-list-releases`.
+
+1. Define helper functions
+
+   ```bash
+   $ die()
+   {
+       echo >&2 "ERROR: $*"
+       exit 1
+   }
+
+   $ github_get_latest_release()
+   {
+      local url="$1"
+
+      local latest=$(curl -sL "$url" | jq -r '.[].tag_name | select(contains("-") | not)' | sort -t "." -k1,1n -k2,2n -k3,3n | tail -1 || true)
+      [ -z "$latest" ] && die "Cannot determine latest release from $url"
+
+      echo "$latest"
+   }
+
+   $ github_get_release_url()
+   {
+      local url="$1"
+      local version="$2"
+
+      download_url=$(curl -sL "$url" | jq --arg version "$version" -r '.[] | select(.tag_name == $version) | .assets[0].browser_download_url' || true)
+      [ -z "$download_url" ] && die "Cannot determine download URL for version $version ($url)"
+
+      local arch=$(uname -m)
+      [ "$arch" = x86_64 ] && arch="($arch|amd64)"
+      echo "$download_url" | egrep -q "$arch" || die "No release for '$arch architecture ($url)"
+
+      echo "$download_url"
+   }
+
+   $ github_download_latest_release()
+   {
+       local url="$1"
+
+       local tmpdir=$(mktemp -d)
+       pushd "$tmpdir" >/dev/null
+
+       local latest=$(github_get_latest_release "$url")
+       local download_url=$(github_get_release_url "$url" "$latest")
+
+       curl -sLO "$download_url"
+       local filename=$(echo "$download_url" | awk -F\/ '{print $NF}')
+
+       ls -d $PWD/${filename}
+
+       popd >/dev/null
+   }
+   ```
+
+1. Perform initial checks
+
+   Check to see if Kata Containers is already installed:
+
+   ```bash
+   $ command -v kata-runtime &>/dev/null && die "Please remove existing Kata Containers installation"
```

Comment:
Checking for kata-runtime will work well for 1.x branches, but if we want to make this bullet proof, checking for the containerd-shim-kata-v2 binary could be a better call.

Reply:
Agreed - done.